Marble Blast Platinum/platinum/client/scripts/loadinggui.cs

```diff
@@ -473,10 +494,13 @@
 $Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: Winterfest was developed in 3 days! It drained our energy very quickly, as bugs were patched only days after released.";
 $Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: Frightfest was developed a week before its first launch. We learned nothing from the Winterfest event.";
 $Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: PlatinumQuest's Ice friction is 0.07331 in value, which is Matan's way of putting leet speek in backwards. MEMES!";
-$Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: PlatinumQuest has seen three seperate development teams throughout its existence. One from 2007 to 2012, one from 2015 to 2021, and one from 2021 to now.";
-$Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: Due to a bug, it used to be possible to submit a time instead of a score on Hunting Around. This would award you so much rating that it hit the 32-bit integer limit. Removing this time reset your total rating to be the same as only your rating from your original Hunting Around score.";
-$Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: Submitting a World Record as '@everyone' on the Marble Blast Platinum Web Port used to have... Unforseen consequences.";
-$Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: Upbeat Finale is actually part of a beta version of Upbeat Freedom. The full version has been lost to time.";
+$Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: PlatinumQuest has seen three seperate development teams throughout its existence. One from 2007 to 2012, one from 2015 to 2021, and one from 2021 to now."; //Yoshi
+$Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: Due to a bug, it used to be possible to submit a time instead of a score on Hunting Around. This would award you so much rating that it hit the 32-bit integer limit. Removing this time reset your total rating to be the same as only your rating from your original Hunting Around score."; //Yoshi
+$Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: Submitting a World Record as '@everyone' on the Marble Blast Platinum Web Port used to have... Unforseen consequences."; //Yoshi
+$Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: Upbeat Finale is actually part of a beta version of Upbeat Freedom. The full version has been lost to time."; //Yoshi
+$Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: A few custom levels brought officially into PlatinumQuest used to have custom music. Commonly used tracks are made by Owen Maitzen (aka hPerks) and sonata1219 (aka NaCl586)."; //Yoshi
+$Loading::Tip[2, $Loading::Tips[2] ++] = "Trivia: One gamemode scrapped in development was called 'Free World', and just served as an exploration gamemode with no objectives and no way to finish. Why not just explore the level anyway? Who needs a specific gamemode for it?"; //Yoshi
```

Comment:
Maybe remove `Why not just explore the level anyway? Who needs a specific gamemode for it?` since that seems too opinion-heavy for a trivia, and the trivia alone will probably lead people to the same conclusion anyway

Reply:
It might also not, though. I could shorten it to "Why not explore levels anyway without a specific gamemode for it?" though